draft-vvv-webtransport-overview.md

```diff
@@ -349,16 +356,14 @@ the network addresses that would normally be inaccessible.  Because of that, the
 user agent that runs untrusted clients MUST NOT provide any detailed error
 information until the server has confirmed that it is a WebTransport endpoint.
 For example, the client must not be able to distinguish between a network
-address that is unreachable and that is reachable but is not a WebTransport
+address that is unreachable and one that is reachable but is not a WebTransport
 server.
 
-WebTransport does not support any traditional means of browser-based
+WebTransport does not support any traditional means of HTTP-based
```

Comment:
I think "HTTP-based authentication" does not actually include all of the things that we want to not support (e.g. prompt-based client auth).

Reply:
What do you mean by "prompt-based client auth" ? What non-HTTP-based authentications are there?

Reply:
TLS client auth (the one where user gets a prompt).

Reply:
WebTransport runs over TLS or QUIC so it has no reason not to support TLS client auth?